### ISSUE TITLE
Scala.js: fix signatures of `SttpFile.fromDomFile` and `sttpFile.toDomFile`

### ISSUE THREAD

**Issue by kamilkloch** [softwaremill/sttp#699]
Currently `SttpFile.fromDomFile` expects `sttp.client.dom.experimental.File` instead of `org.scalajs.dom.raw.File`.
Similarly, `sttpFile.toDomFile` returns `sttp.client.dom.experimental.File` instead of `org.scalajs.dom.raw.File`.
(https://github.com/softwaremill/sttp/blob/master/core/src/main/scalajs/sttp/client/internal/SttpFileExtensions.scala).

Manual casting does the trick and works: 
```scala
SttpFile.fromDomFile(domFile.asInstanceOf[sttp.client.dom.experimental.File])
```
Perhaps it is even an appropriate solution, but I have no idea. 

Perhaps @guymers (who ported stt to Scala.js, thanks!) could take a look?

**Comment by kamilkloch** [comment]
Thank you, @guymers!

**Comment by adamw** [comment]
Released in [2.2.9](https://github.com/softwaremill/sttp/releases/tag/v2.2.9)
